manim/__main__.py

```diff
@@ -123,7 +124,9 @@ def get_scene_classes_from_module(module):
 def get_module(file_name):
     if file_name == "-":
         module = types.ModuleType("input_scenes")
+        logger.info("Enter the code for your animation and end with an EOF (usually CTRL+D):")
```

Comment:
<kbd>Ctrl</kbd> + <kbd>Z</kbd> for Windows.

Reply:
Thanks! I'll be sure to add that in my next commit which should be in a few hours, after I wake up.

Reply:
Done.